src/main/java/no/nav/familie/integrasjoner/dokarkiv/DokarkivService.kt

```diff
@@ -46,17 +46,23 @@ class DokarkivService(private val dokarkivClient: DokarkivClient,
 
         val metadata = dokarkivMetadata.getMetadata(arkiverDokumentRequest.dokumenter[0])
         val arkivdokumenter = arkiverDokumentRequest.dokumenter.map(this::mapTilArkivdokument)
+        val jpsak: Sak?= if(arkiverDokumentRequest.fagsakId!= null) Sak(
+                fagsakId = arkiverDokumentRequest.fagsakId,
```

Comment:
I see you use space before = inconsistently. You should reformat your code before commit. 
There is code style and inspection setting in the project familie-settings in case you haven't imported those.

Reply:
Reformatted, thank you for letting me know the settings repository👍